keyboards/db/db63/README.md

```diff
@@ -0,0 +1,21 @@
+# DB63
+
+A 65% keyboard hotswap with RGB that runs ps2avrgb natively, with USB C, RGB underglow and backlight.
+Firmware is still in **beta**
+
+* Keyboard Maintainer: QMK Community
+* Hardware Supported: 
+* Hardware Availability: 
```

Comment:
If the PCB is for sale anywhere, a link to the product page should go here. Something like:

```suggestion
* Hardware Availability: [DB Store](https://dbstore.vn/)
```

Otherwise, delete this line.

Reply:
The board right now is out of stock, maybe i will remove this, thanks